### DISCUSSION TITLE
Why this code is just showing black screen to my band. Can you please help me

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#116] [Q&A] [answered]
What are my errors. Please help me .at first Answer 1 was working flawlessly but when I add Answer 2 and 3 it began by showing black screen. Please help me by pasting in answer the correct code please.

function randomNum(lowerInteger, upperInteger) {
                        	 let choices = upperInteger - lowerInteger + 1;
                        	 return Math.floor(Math.random() * choices + lowerInteger);
                        }
                        var Answer3=["Um sorry 😔","Sorry "]                                      
var Answer2=["A human ."," made "]
                        var Answer1=["I am  calledra, how can i b at service.","I am s you?","I am "]
                     
              const Answer1 =   hmUI.createWidget(hmUI.widget.BUTTON, {
                            x: 10,
                            y: 100,
                            text: 'you',
                            w: 120,
                            h: 50,
                            radius: 20,                           normal_color: 0x195aea,
                            press_color: 0xffffff,
                            click_func: ()=>{
                            hmUI.showToast({
                                    text: Answer1[randomNum(1, Answer1.length)]
                                  })
                            
                            }
                          })
                  const Answer2 = hmUI.createWidget(hmUI.widget.BUTTON, {
                            x: 10,
                            y: 160,
                            text: 'ou?',
                            w: 120,
                            h: 50,
                            radius: 20,                           normal_color: 0x195aea,
                            press_color: 0xffffff,
                            click_func: ()=>{
                            hmUI.showToast({
                                    text: Answer2[randomNum(1, Answer2.length)]
                                  })
                            
                            }
                          })
                          hmUI.createWidget(hmUI.widget.BUTTON, {
                            x: 10,
                            y: 220,
                            text: 'iend?',
                            w: 120,
                            h: 50,
                            radius: 20,                           normal_color: 0x195aea,
                            press_color: 0xffffff,
                            click_func: ()=>{
                            hmUI.showToast({
                                    text: Answer3[randomNum(2, Answer3.length)]
                                  })
                            
                            }
                          })

**Comment by Rafucho25** [comment, marked as answer]
Hi, you are declaring the variable Answer1 and Answer2 twice:
```javascript
var Answer1=["I am calledra, how can i b at service.","I am s you?","I am "] 
```
and 
```javascript
const Answer1 =   hmUI.createWidget(hmUI.widget.BUTTON, {
```

this is not possible. I recommend using the simulator or check the log in the zepp app. This code should work:

```javascript

    function randomNum(lowerInteger, upperInteger) {
      let choices = upperInteger - lowerInteger + 1;
      return Math.floor(Math.random() * choices + lowerInteger);
    }
  
    var Answer3=["Um sorry ","Sorry "]
    var Answer2=["A human ."," made "]
    var Answer1=["I am calledra, how can i b at service.","I am s you?","I am "]
      
    const BtnAnswer1 = hmUI.createWidget(hmUI.widget.BUTTON, {
      x: 10,
      y: 100,
      text: 'you',
      w: 120,
      h: 50,
      radius: 20,
      normal_color: 0x195aea,
      press_color: 0xffffff,
      click_func: ()=>{
      hmUI.showToast({
          text: Answer1[randomNum(1, Answer1.length)]
        })
      }
    })

    const BtnAnswer2 = hmUI.createWidget(hmUI.widget.BUTTON, {
      x: 10,
      y: 160,
      text: 'ou?',
      w: 120,
      h: 50,
      radius: 20,                           
      normal_color: 0x195aea,
      press_color: 0xffffff,
      click_func: ()=>{
        hmUI.showToast({
          text: Answer2[randomNum(1, Answer2.length)]
        })
      }
    })

    hmUI.createWidget(hmUI.widget.BUTTON, {
      x: 10,
      y: 220,
      text: 'iend?',
      w: 120,
      h: 50,
      radius: 20,
      normal_color: 0x195aea,
      press_color: 0xffffff,
      click_func: ()=>{
        hmUI.showToast({
          text: Answer3[randomNum(2, Answer3.length)]
        })
      }
    })
```

![image](https://user-images.githubusercontent.com/16562078/225334078-830032d6-a83a-46c3-882b-ce5f5ceaeff1.png)

**Reply by alexmathew11** [reply]
Thank you sooooo much man you are the first person who helped me correctly and genuinely at this platform . I am having only one word for your bravo👏
